source/_components/switch.xiaomi_vacuum.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Xiaomi Mi Robot Vacuum"
+description: "Instructions how to integrate your Xiaomi Mi Robot Vacuum within Home Assistant."
+date: 2017-05-05 18:11
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi_vacuum.png
+ha_category: Switch
+ha_release: 0.47
+---
+
+The `xiaomi_vacuum`switch platform allows you to control the state of your [Xiaomi Mi Robot Vacuum](http://www.mi.com/roomrobot/). 
+Current supported features are start and stop (goes to dock).
+
+{% linkable_title Getting started %}
+
+To be able to communicate with the vacuum one needs to have its IP address as well as an encryption token.
+This token is only attainable before the device has been connected over the app to your local wifi.
```

Comment:
This sentence would be clearer as:
`You must retrieve the token from the device before using the mobile app to connect it to your wireless network.`